welcomescreen.py

```diff
@@ -34,6 +34,9 @@ class welcomescreen:
 
     def __init__(self, display):
         self.gameDisplay = display
+        self.x1_x2 = [[350, 350 + 490] for _ in range(4)]
```

Comment:
How did you find these numbers, are they by trial and error? Also, do you think if moving these numbers to variables might help?

Reply:
Unfortunately so many numbers are hardcoded everywhere already in the activity, 490 being the width of actual game. These numbers were already there in the form of `land1x = 350` and `land2x = 840` which I just converted in form of array.

Reply:
I extracted hardcoded values and also added more comments.